src/components/MoneyRequestHeader.tsx

```diff
@@ -231,7 +244,32 @@ function MoneyRequestHeader({
                     policy={policy}
                     shouldShowBackButton={shouldUseNarrowLayout}
                     onBackButtonPress={onBackButtonPress}
-                />
+                >
+                    {shouldShowMarkAsCashButton && !shouldUseNarrowLayout && (
+                        <Button
+                            success
+                            medium
+                            text={translate('iou.markAsCash')}
+                            style={[styles.p0]}
+                            onPress={() => {
+                                TransactionActions.markAsCash(transaction?.transactionID ?? '', transactionThreadReportID ?? '');
+                            }}
+                        />
+                    )}
+                </HeaderWithBackButton>
+                {shouldShowMarkAsCashButton && shouldUseNarrowLayout && (
+                    <View style={[styles.ph5, styles.pb3]}>
+                        <Button
+                            medium
+                            success
+                            text={translate('iou.markAsCash')}
+                            style={[styles.w100, styles.pr0]}
+                            onPress={() => {
+                                TransactionActions.markAsCash(transaction?.transactionID ?? '', transaction?.reportID ?? '');
```

Comment:
```suggestion
                                TransactionActions.markAsCash(transaction?.transactionID ?? '', report?.reportID ?? '');
```

I think it is better to extract this to a local function because it is used twice and in the last commit only one instance was updated.